src/handlers/motions/motionVoteRevealed/helpers.ts

```diff
@@ -0,0 +1,42 @@
+import { BigNumber } from 'ethers';
+
+import { MotionMessage, MotionEvents } from '~types';
+
+interface Props {
+  messages: MotionMessage[];
+  messageKey: string;
+  voter: string;
+  votes: [BigNumber, BigNumber];
+}
+
+export const getUpdatedMessages = ({
+  messages,
+  messageKey,
+  voter,
+  votes: [nayVotes, yayVotes],
+}: Props): MotionMessage[] => {
+  const updatedMessages = [
+    ...messages,
+  ];
+
+  if (nayVotes.gte(yayVotes)) {
```

Comment:
I would sync these with the motion state via the lamda, else it's going to publish this message every time someone reveals their vote.

Reply:
Moved to cdapp fetchMotionState lambda